packages/react-ui-core/src/Photo/Photo.js

```diff
@@ -75,23 +106,31 @@ export default class Photo extends PureComponent {
       url: _,
       alt,
       fallbackUrl,
+      sources,
       ...rest
     } = this.props
 
     const { url, error } = this.state
 
+    const imageTag = (<img
+      src={error ? fallbackUrl : url}
+      alt={alt}
+      data-tid="photo"
+      className={classnames(
+        className,
+        theme.Photo
+      )}
+      onError={this.fallback}
+      {...rest}
+    />)
+
+    if (error || !this.sourceTags) return imageTag
```

Comment:
we want to return an imageTag if there is an error?

Reply:
this ensures the fallback (or nothing) is displayed, otherwise it will try to select a "better" source from the `srcset`, even if it doesn't exist, and you end up with this:

![screen shot 2018-07-20 at 8 59 50 am](https://user-images.githubusercontent.com/1353235/43012576-51e735fc-8bfb-11e8-93b9-d76826f1ddf3.png)

Reply:
Since we have no fallback on Rent at this time, the desired outcome is: 

![screen shot 2018-07-20 at 9 02 55 am](https://user-images.githubusercontent.com/1353235/43012725-c254052c-8bfb-11e8-8e02-6a40e6a66bff.png)